dist/common/scripts/scylla_util.py

```diff
@@ -359,8 +359,8 @@ class azure_instance:
     EPHEMERAL = "ephemeral"
     ROOT = "root"
     GETTING_STARTED_URL = "http://www.scylladb.com/doc/getting-started-azure/"
-    META_DATA_BASE_URL = "http://169.254.169.254/1.0/meta-data/instance/"
-    ENDPOINT_SNITCH = "GossipingPropertyFileSnitch"
+    META_DATA_BASE_URL = "http://169.254.169.254/metadata/instance"
+    ENDPOINT_SNITCH = "SimpleSnitch"
```

Comment:
I still don't agree with SimpleSnitch , it will create us more problems when people will start opening cases how to move from it to GossippingProperty , if azure images will use it
we should either use GossippingProperty and pass on params later
or implement AzureSnitch https://github.com/scylladb/scylla/issues/8593

Reply:
Sorry forgot to revert it, just used it for debugging , will send a fix later

Reply:
Fixed